bulbs/content/tasks.py

```diff
@@ -51,3 +63,117 @@ def update_feature_type_rates(featuretype_pk):
                 rate=0,
                 feature_type_id=featuretype_pk,
                 role_id=role.pk)
+
+
+@shared_task(default_retry_delay=5)
+def post_to_instant_articles_api(content_pk):
```

Comment:
This is a pretty huge function, i recommend breaking logic into 2 helper methods (add + delete)